jvb/src/main/kotlin/org/jitsi/videobridge/colibri2/Colibri2ConferenceHandler.kt

```diff
@@ -71,7 +73,7 @@ class Colibri2ConferenceHandler(
 
         /* TODO: is there any reason we might need to handle Endpoints and Relays in in-message order? */
```

Comment:
Your validation I think now means this TODO can go away, because we don't support it.